src/components/Items/Items.jsx

```diff
@@ -3,7 +3,6 @@ import React from 'react';
 
 export default (props) => {
   return <div className="Menu_Items">
-    <Items name={props.items.name} />,
-    <Items price={props.items.price} />
+    <p> {props.name}, {props.price} </p>
```

Comment:
I guess what I'm not getting as when to use a react tag and when to use a regular html tag